test/gw/likelihood/marginalization_test.py

```diff
@@ -231,8 +231,8 @@ def setUp(self):
 
         trial_roq_paths = [
             "/roq_basis",
-            os.path.join(os.path.expanduser("~"), "ROQ_data/IMRPhenomPv2/4s"),
```

Comment:
Here, I don't think we have ROQ data for XPHM yet, so this will need to be reverted (and may be why the CI is failing).

Reply:
@mj-will I assume #897 was a "move towards" modern waveforms and it is okay to use the older waveforms for the ROQs to avoid having to figure out how to package them?

Reply:
Yes, I think that's fine. 

I'll note there is definitely ROQ data for XPHM on CIT but I don't think it's public (and is also huge).